dspace-spring-rest/src/main/java/org/dspace/app/rest/Application.java

```diff
@@ -47,7 +48,7 @@
  * @author Andrea Bollini (andrea.bollini at 4science.it)
  * @author Tim Donohue
  */
-@SpringBootApplication
+@SpringBootApplication(exclude = VelocityAutoConfiguration.class)
```

Comment:
@mwoodiupui : This exclusion should be moved to our `application.properties` file alongside all the other AutoConfiguration exclusions.  See this section of that config: https://github.com/DSpace/DSpace/blob/master/dspace-spring-rest/src/main/resources/application.properties#L80-L93

Reply:
Thank you, that makes sense.  I did not know about `application.properties:spring.autoconfigure.exclude`.